akka-http-core/src/test/scala/akka/http/javadsl/JavaInitializationSpec.scala

```diff
@@ -19,46 +19,40 @@ class JavaInitializationSpec extends WordSpec with Matchers {
     "initializes the right field" in {
       akka.http.scaladsl.model.headers.EntityTagRange.`*` =!= "*"
       akka.http.javadsl.model.headers.EntityTagRanges.ALL =!= "*"
-      akka.http.javadsl.model.headers.EntityTagRange.ALL =!= "*"
```

Comment:
:+1:

Weird that they weren't all in the same order because the bug so much depended on the order of initialization...